build_tools/python/e2e_test_framework/models/torch_models.py

```diff
@@ -0,0 +1,52 @@
+## Copyright 2022 The IREE Authors
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+"""Defines PyTorch models."""
+
+from e2e_test_framework import unique_ids
+from e2e_test_framework.definitions import common_definitions
+
+# Implementations of the models listed below can be found in `https://github.com/iree-org/iree-samples/tree/main/iree-torch/importer`.
+# We import the PyTorch models offline and make the .mlir available here for benchmarking.
+# If the mlir artifacts need to be updated, please run [update_torch_models.sh](https://github.com/iree-org/iree-samples/blob/main/iree-torch/importer/update_torch_models.sh)
+# Then update the `source_url` below with the new paths.
```

Comment:
nits:
* Only wrap hyperlinks in `` if you _don't_ want the URLs to be clickable, e.g. `www.example.com`
* No need to use markdown `[link text](www.link_url.com)` in a code comment, unless this comment is going to be rendered as markdown somewhere

```suggestion
# Implementations of the models listed below can be found in
# https://github.com/iree-org/iree-samples/tree/main/iree-torch/importer.
# We import the PyTorch models offline and make the .mlir available here for benchmarking.
# If the mlir artifacts need to be updated, please run
# https://github.com/iree-org/iree-samples/blob/main/iree-torch/importer/update_torch_models.sh
# Then update the `source_url` below with the new paths.
```

Reply:
Thanks, updated.